lib/tasks/update_resources.rake

```diff
@@ -35,8 +35,7 @@ task update_resources: :environment do
     #=============================
     #   update the database      #
     #=============================
-    type_names = ['Wireless Bodypack Transmitter', "Instructional PC Desktop",
-                  "Blu-Ray Player", "Lecture Capture Camera", "Video Projector"]
+    type_names = AppPreference.find_by(name: "resource_types").value.split(",").each(&:strip!)
```

Comment:
Sorry, I didn't think about that when I suggested using resource_types AppPreference. Before running the whole task, shouldn't we check that the preference is not empty and that it has a list of resources? If the preference is empty, there is no need to run the task. We can create a note in the RoomUpdateLog table asking admins to add a list of resource types.